SostavSD/Pages/Contracts/ContractListTable.razor

```diff
@@ -1,42 +1,48 @@
-﻿<div class="table-responsive">
-    <table class="table table-striped">
-        <thead class="thead-light">
-            <tr>
-                <th>ProjectName</th>
-                <th>Index</th>
-                <th>Order</th>
-                <th>ContractNumber</th>
-                <th>ContractDate</th>
-                <th>ContractDateEndOfWork</th>
-                <th>City</th>
-            </tr>
-        </thead>
-
-
-        <tbody>
-            @if (_contracts == null)
-            {
-                <tr>Loading...</tr>
-            }
-            else if (_contracts.Count == 0)
-            {
-                <tr>No items</tr>
-            }
-            else
-            {
-                @foreach (var item in _contracts)
-                {
-                    <tr>
-                        <td>@item.ProjectName</td>
-                        <td>@item.Index</td>
-                        <td>@item.Order</td>
-                        <td>@item.ContractNumber</td>
-                        <td>@item.ContractDate.ToShortDateString()</td>
-                        <td>@item.ContractDateEndOfWork.ToShortDateString()</td>
-                        <td>@item.City</td>
-                    </tr>
-                }
-            }
-        </tbody>
-    </table>
-</div>
\ No newline at end of file
+﻿@inject ISnackbar Snackbar
+@inject IDialogService _dialogService
+
+<MudContainer Class="d-flex justify-center mb-sm-5">
+	<MudFab Color="Color.Primary" Icon="@Icons.Material.Filled.Add" Size="Size.Small" Label="Новый договор" @onclick="(e => CreateNewContract())">
+
+	</MudFab>
+</MudContainer>
+
+<MudTable Items="@_contracts" Filter="new Func<ContractModel,bool>(FilterFunc1)" @bind-SelectedItem="selectedItem">
+
+	<ToolBarContent>
+		<MudSpacer />
+		<MudTextField @bind-Value="searchString" Placeholder="Поиск" Adornment="Adornment.Start" AdornmentIcon="@Icons.Material.Filled.Search" IconSize="Size.Medium" Class="mt-0"></MudTextField>
+	</ToolBarContent>
+	<HeaderContent >
+
+		<MudTh>Объект</MudTh>
+		<MudTh>Индекс</MudTh>
+		<MudTh>Заказ</MudTh>
+		<MudTh>Номер договора</MudTh>
+		<MudTh>Дата договора</MudTh>
+		<MudTh>Дата окончания работ</MudTh>
+		<MudTh>Город</MudTh>
+
+	</HeaderContent>
+
+	<RowTemplate>
+		<MudTd DataLabel="Объект">@context.ProjectName</MudTd>
+		<MudTd DataLabel="Индекс">@context.Index</MudTd>
+		<MudTd DataLabel="Заказ">@context.Order</MudTd>
+		<MudTd DataLabel="Номер договора">@context.ContractNumber</MudTd>
+		<MudTd DataLabel="Дата договора">@Convert.ToDateTime(context.ContractDate).ToString("dd/MM/yyyy")</MudTd>
```

Comment:
Create a separate class for Constants like "dd/MM/yyyy"
It will be easier to change it in one place for the whole application